README.adoc

```diff
@@ -305,3 +305,13 @@ behind as you generate.
       </plugin>
 ----
 
+=== Testing
+
+To test locally you can run tests by invoking this command:
+
+[source,bash]
+----
+mvn -Dmaven.wagon.http.retryHandler.count=5 --batch-mode verify
```

Comment:
this should be `mvn clean verify` - the other stuff is if you have a clean build and unreliable build process like GitHub is :-(